CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+## 5.2.0 - *2023-05-26*
```

Comment:
Please remove this line, it'll automatically get updated by our system.

Reply:
Done!